addon/components/frost-list.js

```diff
@@ -117,9 +117,11 @@ export default Component.extend({
   // == Functions =============================================================
 
   setShift (event) {
-    if (!this.isDestroyed) {
-      this.set('_isShiftDown', event.shiftKey)
-    }
+    run.next(() => {
```

Comment:
Previously holding shift while the test would run would cause errors. This fixes that.